smarts/core/vehicle.py

```diff
@@ -50,6 +53,16 @@
 from .utils.math import rotate_around_point
 
 
+class ActorRole(IntEnum):
+    """Used to specify the role an actor (e.g. vehicle) is currently playing in the simulation."""
+
+    Unknown = 0
```

Comment:
I don't think it's that problematic though because nobody should be using an enum value as a boolean anyway.  On the other hand, I have often seen `IntEnum` values used as bitmasks.  And although we aren't doing so here (yet), I think it makes sense of `Unknown` to be 0 to keep this possibility as an option.